test/index.test.js

```diff
@@ -6,6 +6,7 @@ const path = require("path");
 const webpack = require("webpack");
 const fs = require("fs");
 const merge = require("webpack-merge");
+const sass = require("node-sass");
```

Comment:
Tests should be running on `node-sass` an `dart-sass` together, we need change `test` script:
1. Install `node-sass` and run tests
2. Install `dart-sass` and run tests.

Reply:
I'm not sure how to do that gracefully, especially if the implementation isn't supplied as an option.

Reply:
@nex3 skip tests where `dart-sass` doesn't have options/features/etc and add comments about this. Why? After release all people starting upgrade and we will get a lot of issues like "It is woks on `x.x.x` version, but after upgrade to `x.x.x` it is doesn't works. [A lot of angry words]" (a lot of people don't read changelog and release pages). We should prepare to this.

Reply:
Okay, but at a higher level I'm not sure how to run these tests with both versions of Sass or how to switch between those versions.

Reply:
Just add additional look here https://github.com/webpack-contrib/sass-loader/blob/master/test/index.test.js#L35 with `node-sass` and `sass` values and we need some modify tests to allow using difference implementation

Reply:
I'm still not sure I understand. Can you give me an example of what a test you're envisioning would look like?

Reply:
Like this (pseudo code):
```js
implementations.forEach(implementation => {
  syntaxStyles.forEach(ext => {
    loaderOptions.implementation = implementation
    // tests
  });
})
```

Reply:
Also add `implementations.forEach` here https://github.com/webpack-contrib/sass-loader/blob/master/test/index.test.js#L112